### PR TITLE
docs: fix small typos and stylings in setup.md

### PR DESCRIPTION
**Pull request by NeroBurner**
#### Summary
Fix some small typos in the documentation for the setup, and clarify formulations.

In the "Menu Bar" documentation unify on the imperative (do vs does)

#### Ticket Link
no link

#### Checklist

- [ ] Added or updated unit tests (required for all new features)
- [ ] Has UI changes
- [x] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
- [x] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
- [ ] executed `npm run lint:js` for proper code formatting
  - no, didn't touch any code, just documentation

#### Device Information
This PR was tested on: None

#### Screenshots
None

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ To set up the Mattermost desktop application for Windows 7, 8 and 10:
 
 
 ### OS X
-For OS X 10.11 El Capitan. An older version of OS X has similar way.
+For OS X 10.11 El Capitan. For older versions of OS X the instruction are similar.
```
Comment (justinegeffen) [inline docs/setup.md:29]:
```suggestion
For macOS 10.11 El Capitan. For older versions of macOS the instruction are similar.
```

File: docs/setup.md [hunk 2]
```diff
@@ -65,17 +65,17 @@ For Ubuntu 16.04.
...
 
-    Please refer https://wiki.archlinux.org/index.php/Desktop_entries
+    Please refer to https://wiki.archlinux.org/index.php/Desktop_entries
 
 ## Configuration
 You have to configure the application to interact with your teams.
 
 1. Open **Settings Page**. If you use the application for the first time, **Settings Page** should appear automatically.
-   You can also see **Settings Page** by following methods.
+   You can also access the **Settings Page** by the following methods:
```
Comment (justinegeffen) [inline docs/setup.md:78]:
```suggestion
   You can also access the **Settings Page** using one of the following methods:
```

File: docs/setup.md [hunk 3]
```diff
@@ -65,17 +65,17 @@ For Ubuntu 16.04.
...
 
    - Windows: Press `Alt` key to bring up the menu at the top of the window, then click `File -> Settings`.
```
Comment (justinegeffen) [inline docs/setup.md:80]:
```suggestion
   - Windows: Press the `ALT` key to bring up the menu at the top of the window, then select **File > Settings**.
```

File: docs/setup.md [hunk 4]
```diff
@@ -65,17 +65,17 @@ For Ubuntu 16.04.
...
    - OS X: Click `Mattermost` from the menu at the top of the screen, then click `Preferences...`.
```
Comment (justinegeffen) [inline docs/setup.md:81]:
```suggestion
   - macOS: Select **Mattermost** from the menu at the top of the screen, then select **Preferences**.
```

File: docs/setup.md [hunk 5]
```diff
@@ -121,7 +121,7 @@ The Settings Page is available from the **File** menu under **Settings** (Click
    - **Hide Menu Bar** (Windows, Linux)
       - This option hides the menu bar. Press "Alt" to show it.
    - **Show Icon on Menu Bar** (OS X)
```
Comment (justinegeffen) [inline docs/setup.md:123]:
```suggestion
   - **Show Icon on Menu Bar** (macOS)
```

File: docs/setup.md [hunk 6]
```diff
@@ -120,69 +121,66 @@ The Settings Page is available from the **File** menu under **Settings** (Click
...
+  - **Toggle Developer Tools** (CTRL+SHIFT+I) - Turn on and off sidebar showing developer tools
   - **Actual Size** (Ctrl+0) - Reset zoom level
-  - **Zoom In** (Ctrl+Plus) - Enlarge the rendered contents size
-  - **Zoom In** (Ctrl+Minus) - Shrink the rendered contents size
+  - **Zoom In** (CTRL++) - Enlarge the rendered contents size
+  - **Zoom In** (CTRL+-) - Shrink the rendered contents size
 - **History**
-  - **Back** (Alt+Left, Command+[ on OS X) - Go back to previous web page in the current tab
-  - **Forward** (Alt+Right, Command+] on OS X) - Go forward to next web page in the current tab
+  - **Back** (ALT+LEFT, CMD+`[` on macOS) - Go back to previous web page in the current tab
```
Comment (justinegeffen) [inline docs/setup.md:166]: Please could you confirm this command as it looks similar to the one below. Thank you!

File: docs/setup.md (hunk 5 above)
Comment (NeroBurner) [reply docs/setup.md]: can't apply from GitHub itself, the changed line is "Outdated"

File: docs/setup.md (hunk 6 above)
Comment (NeroBurner) [reply docs/setup.md:166]: ALT+LEFT and ALT+RIGHT work on Linux, got no Mac to test this on

Comment (justinegeffen) [reply docs/setup.md:166]: Thanks for testing! I tested on macOS and can confirm the listed command works. I just changed the formatting. :)

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thanks for this PR, @NeroBurner! Since it was submitted, this file has been deleted from the repo. Unfortunately this means the PR can't be merged and I'll be closing it. Thank you so much for your time and effort - we really appreciate it. :)

**Comment by NeroBurner** [comment]
you're welcome, and thanks for this awesome chat application

**Pull request closed without merging**